Description: Applied GCC hardening flags to build process.
Author: Alexandre Dantas <eu@alexdantas.net>
Last-Update: 2014-08-18
--- xbomb.orig/Makefile
+++ xbomb/Makefile
@@ -32,12 +32,12 @@ INSTDIR=/usr/local
 ########
 
 xbomb : $(OBJ)
-	$(LINK) $(OBJ) -o $@ $(LIB) $(XLIB)
+	$(LINK) $(OBJ) -o $@ $(LIB) $(XLIB) $(LDFLAGS)
 
 ########
 
 %.o : %.c
-	$(COMPILE) $< -o $@ $(INCLUDES)
+	$(COMPILE) $(CPPFLAGS) $(CFLAGS) $(CXXFLAGS) $< -o $@ $(INCLUDES)
 
 xbomb.o   : xbomb.c   xbomb.h
 xwindow.o : xwindow.c xbomb.h icon.h
